blackbox/docs/configuration.txt

```diff
@@ -403,7 +403,7 @@ Ports
   of the node is not directly reachable from outside, e.g. running it
   behind a firewall or inside a Docker container.
 
-.. _psql.port:
+.. _psql_port:
```

Comment:
yeah this was not about that, you can't link directly to this for whatever reason :(